src/app.ts

```diff
@@ -17,6 +17,8 @@ app.use(bodyParser.urlencoded({
   extended: true
 }))
 
+app.use(express.static('views'));
```

Comment:
so I would take a look at those express docs again - it's using a directory "public". I would follow that pattern so move all these images out of views :) 

and then we defo want something like this app.use(express.static(path.join(__dirname, 'public'))) so we can always find the folder.

Note in the docs -> it has /static -> that defines a url where the images can be found. If you go localhost:3000/images/Flux_White_Footer.png right now it'll load. I don't hate putting it all under static. So maybe just copy whats in those docs

Reply:
this caused so much more drama than I thought it would... will resolve @erinmcj